File: LES_SERIE_S1_IA_FS_ALGO_PYTHON_C/Série_de_TD-TP_3/exercice8.py
N_saisies=0
while True:
    N=float(input("saisir un nombre: "))
    N_saisies+=1
    user_choice=str(input("saisir non ou Non pour quite: "))
    if(user_choice=="non" or user_choice=="Non"):
        break
print("le nombre de valeurs saisies est: ",N_saisies)

File: LES_SERIE_S1_IA_FS_ALGO_PYTHON_C/Série_de_TD-TP_3/exercice8.c
#include <stdio.h>
#include <string.h>
int N_saisies;
float N;
char user_choice[10];
int main(){
    N_saisies=0;
    do{
        printf("saisir un nombre: ");
        scanf("%f",&N);
        N_saisies++;
        printf("saisir 'non' ou 'Non' pour quite.\n");
        scanf("%s",&user_choice);
    }while(strcmp(user_choice, "non")!=0 && strcmp(user_choice, "Non")!=0);
    printf("le nombre de valeurs saisies est: %d",N_saisies);
    return 0;
}